doc/documents/conf.py

```diff
@@ -86,7 +86,7 @@
 todo_include_todos = True
 
 #If true, figures and tables are numbered
-numfig = True
+numfig = False
```

Comment:
This is comment to line 66 above.
Change version = u'1.00a' to version = u'1.00'

Reply:
Done it. Please check